DataConnectors/AtlassianJiraAudit/JiraNativePollerConnector/azuredeploy_Jira_native_poller_connector.json

```diff
@@ -0,0 +1,147 @@
+                                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key)",
+                                "providerDisplayName": "Keys",
+                                "scope": "Workspace",
+                                "requiredPermissions": {
+                                    "action": true
+                                }
+                            }
+                        ],
+                        "customs": [
+                            {
+                                "name": "Atlassian Jira API credentials",
+                                "description": "Jira Username and Jira Access Token are required. [See the documentation to learn more about Atlassian Jira API](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/#about). Jira domain must be provided as well."
+                            }
+                        ]
+                    },
+                    "instructionSteps": [
+                        {
+                            "title": "Connect Atlassian Jira",
+                            "description": "Please inset your credentials",
+                            "instructions": [
+                                {
+                                    "parameters": {
+                                        "enable": "true",
+                                        "userRequestPlaceHoldersInput": [
+                                            {
+                                                "displayText": "Domain Name",
+                                                "requestObjectKey": "apiEndpoint",
+                                                "placeHolderName": "{{domain}}",
+                                                "placeHolderValue": ""
+                                            }
+                                        ]
+                                    },
+                                    "type": "BasicAuth"
+                                }
+                            ]
+                        }
+                    ]
+                },
+                "pollingConfig": {
+                    "auth": {
+                        "authType": "Basic"
+                    },
+                    "request": {
+                        "apiEndpoint": "https://{{domain}}/rest/api/3/auditing/record",
+                        "httpMethod": "Get",
+                        "queryTimeFormat": "yyyy-MM-ddTHH:mm:ssZ",
+                        "startTimeAttributeName": "from",
+                        "endTimeAttributeName": "to",
+                        "queryWindowInMin": 5
+                    },
+                    "paging": {
+                        "pagingType": "Offset",
```

Comment:
Value "Offset" is not defined in enum. Please  refer template.

Reply:
It is a new type of pagination. It was developed recently and provided to us by Microsoft team.